src/main/groovy/icaruswings/repositories/PaymentRepository.groovy

```diff
@@ -23,16 +29,22 @@ class PaymentRepository implements Repository<Payment, PaymentRepository> {
             if (search.containsKey("payer")) {
                 eq("payer.id", Long.valueOf(search.payer.toString()))
             }
+
+            if (search.containsKey("paymentStatus[in]")) {
+                inList("paymentStatus", search."paymentStatus[in]")
```

Comment:
normalmente fazemos o in no seguinte formato:
`"in"("type", search."type[in]".collect { BatchType.valueOf(it.toString()) })` 
para garantir a conversão